apps/ejabberd/src/muc_admin.erl

```diff
@@ -0,0 +1,100 @@
+%%==============================================================================
+%% Copyright 2016 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% Author: Joseph Yiasemides <joseph.yiasemides@erlang-solutions.com>
+%% Description: Administration commands for Mult-user Chat (MUC)
+%%==============================================================================
+
+-module(muc_admin).
+
+-behaviour(gen_mod).
+-export([start/2, stop/1]).
+
+-export([create_room/4]).
+-export([invite_to_room/5]).
+
+-include("ejabberd.hrl").
+-include("jlib.hrl").
+
+start(_, _) ->
+    mongoose_commands:register(commands()).
+
+stop(_) ->
+    mongoose_commands:unregister(commands()).
+
+commands() ->
+    [
+     [{name, create_muc_room},
+      {category, muc},
```

Comment:
The category is the later translated to the resource in REST path. The good practise is to use plural nouns for that. I think category name `mucs` will suite better. But I'm open to here a constructive disagreement :)